Cargo.toml

```diff
@@ -52,10 +53,16 @@ std = ["digest/std", "pkcs1/std", "pkcs8/std", "rand_core/std", "signature/std"]
 pem = ["pkcs1/pem", "pkcs8/pem"]
 pkcs5 = ["pkcs8/encryption"]
 getrandom = ["rand_core/getrandom"]
+algorithm-identifier = ["pkcs1/pkcs8"]
```

Comment:
This is already enabled by default? I don't see any need to feature-gate `algorithm-identifier` whatsoever

Reply:
done